### PR TITLE
Tutorial Update

### PR DESCRIPTION
**Pull request by MattToast**
Updates tutorials 01, 03, and 05. Changes include:
- Use and explain `Experiment.create_run_settings` and `Experiment(launcher="auto")`
- Add clarifications to sections that I felt could use it
- Minor typos/spelling/grammar

### CODE REVIEW

**Comment by MattToast** [comment]
Without looking to introduce documentation conflicts with the outstanding changes in #115, I have added some very minor updates for tutorial 04 as well.

File: doc/tutorials/lattice_boltz_analysis.rst [hunk 1]
```diff
@@ -195,6 +203,10 @@ the `Experiment` object, the following can be called to stop any database instan
     # be sure to be in Python environment where SmartSim is installed
     $(smart --dbcli) -h 127.0.0.1 -p 6780 shutdown
```
Comment (Spartee) [inline doc/tutorials/lattice_boltz_analysis.rst:204]: this is just ``smart dbcli`` now

Comment (Spartee) [inline doc/tutorials/lattice_boltz_analysis.rst:204]: this is just ``smart dbcli`` now

File: tutorials/05_starting_ray/05_starting_ray_builtin.ipynb [hunk 2]
```diff
@@ -5,23 +5,25 @@
    "id": "4cba3240",
    "metadata": {},
```
Comment (Spartee) [inline tutorials/05_starting_ray/05_starting_ray_builtin.ipynb:6]: ensure all the metadata for the kernelspec and interpreter are taken out of each notebook. (I usually do this by just manually editing the ipynb file in vim)

**Review by Spartee — APPROVED** [review]
Minor comments to fix, but otherwise LGTM.

File: doc/tutorials/lattice_boltz_analysis.rst (hunk 1 above)
Comment (MattToast) [reply doc/tutorials/lattice_boltz_analysis.rst:204]: Looks like duplicate of last comment, resolving

File: tutorials/05_starting_ray/05_starting_ray_builtin.ipynb (hunk 2 above)
Comment (MattToast) [reply tutorials/05_starting_ray/05_starting_ray_builtin.ipynb:6]: Fixed in c0fb85c

File: doc/index.rst [hunk 3]
```diff
@@ -14,11 +14,10 @@
    :caption: Tutorials
 
    tutorials/01_getting_started/01_getting_started
-   tutorlals/03_online_analysis/03_online_analysis
-   tutorials/05_starting_ray/05_starting_ray
    tutorials/using_clients
    tutorials/lattice_boltz_analysis
    tutorials/inference
+   tutorials/05_starting_ray/05_starting_ray_builtin
```
Comment (al-rigazzi) [inline doc/index.rst:20]: I see we are mixing numbered and not-numbered tutorials, like `01_getting_started` and `inference`. Would it make sense to simply switch to not-numbered ones? I'm concerned the mix could look confusing.

File: doc/tutorials/lattice_boltz_analysis.rst [hunk 4]
```diff
@@ -127,8 +132,9 @@ The next portion starts the database and immediately connects
 a client to it so that data can be retrieved by the analysis
 code and plotted.
 
-The simulation is started with `block=False`, so that the data
-being streamed from the simulation can be analyzed in real time.
+The simulation is started with the `block=False` argument. This runs the simulation
+in a nonblocking manner so that the data can being streamed from the simulation can be
```
Comment (al-rigazzi) [inline doc/tutorials/lattice_boltz_analysis.rst:136]: "can being" -> "being"

File: tutorials/05_starting_ray/05_starting_ray_builtin.ipynb [hunk 5]
```diff
@@ -216,14 +232,14 @@
    "id": "6da5f0a5",
    "metadata": {},
    "source": [
-    "## 3. Stop cluster and release allocation\n",
+    "## 5.3 Stop Cluster and Release Allocation\n",
     "\n",
-    "We first shut down the Ray runtime, then disconnect the context."
+    "When we are finnished with the cluster and ready to deallocate resources, we must first shut down the Ray runtime, followed by disconnecting the context."
```
Comment (al-rigazzi) [inline tutorials/05_starting_ray/05_starting_ray_builtin.ipynb:237]: "finnished"->"finished"

**Review by al-rigazzi — COMMENTED** [review]
Just a few typos, and one thing which looks a bit confusing. Otherwise LGTM

File: doc/tutorials/lattice_boltz_analysis.rst (hunk 4 above)
Comment (MattToast) [reply doc/tutorials/lattice_boltz_analysis.rst]: Fixed in ce97d1d

File: tutorials/05_starting_ray/05_starting_ray_builtin.ipynb (hunk 5 above)
Comment (MattToast) [reply tutorials/05_starting_ray/05_starting_ray_builtin.ipynb]: Fixed in ce97d1d

File: doc/index.rst (hunk 3 above)
Comment (MattToast) [reply doc/index.rst:20]: The contents of the tutorials are number (that is sections in `using_clients` are labeled `2.x`). That said, I do like keeping the naming uniformity for all of the tutorial files. Adding numbers to the rst files

Comment (MattToast) [reply doc/index.rst:20]: Fixed in 43536be

**Pull request merged**